src/main/java/transforms/ConcatCombineTransform.java

```diff
@@ -34,23 +34,19 @@ public ConcatCombineTransform(
     public Object apply( Map<String, String> row ) {
         StringBuilder sb = new StringBuilder();
         String sep = "";
-        for ( Transformation s : transforms )
-        {
+        for ( Transformation s : transforms ) {
             Object toadd = s.apply(row);
-            if ( !(toadd == null)) {
-                if (!StringUtils.isBlank( toadd.toString()) ) {
-                    sb.append(sep).append(toadd);
-                    sep = separator;
-                }
+            if ( toadd != null && !StringUtils.isBlank( toadd.toString()) ) {
```

Comment:
another 0% important thing -- can you camelCase this? In my head this sounds like "toad" at first

Reply:
👍